services/AmazonAlexa.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "AmazonAlexa",
```

According to #106:

```suggestion
  "name": "Alexa",
```

Or, if ambiguous:

```suggestion
  "name": "Amazon Alexa",
```